picoquic/picoquic_internal.h

```diff
@@ -376,6 +378,11 @@ typedef struct st_picoquic_cnx_t {
     unsigned int ack_needed : 1;
     unsigned int current_spin : 1; /* Current value of the spin bit */             
     unsigned int client_mode : 1; /* Is this connection the client side? */
+    unsigned int prev_spin : 1;  /* previous Spin bit */
+    unsigned int spin_vec : 2; 
+    unsigned int spin_edge : 1;
+    uint64_t lastTrigger;
+
```

Comment:
What is the definition of last trigger?

Reply:
It is the timestamp of the incoming packet that triggered the last spin bit change, to be aired on next outgoing packet.  Necessary to detect when the reaction is too slow, and advertise it through VEC==1. Now documented in comments.